compiler-core/src/language_server/engine.rs

```diff
@@ -274,6 +278,7 @@ where
             code_action_fix_names(module, &params, &mut actions);
             actions.extend(LetAssertToCase::new(module, &params).code_actions());
             actions.extend(RedundantTupleInCaseSubject::new(module, &params).code_actions());
+            actions.extend(MoveImportsToTop::new(module, &params).code_actions());
```

Comment:
If you're going to move them to module could you move all of them please. There are still two in this module. 🙏 

This bit here could look like this:

```rust
code_actions::all(module, &params, &actions)
```

Reply:
I'm going back to the single module for now!